test/CMakeLists.txt

```diff
@@ -24,12 +24,12 @@ if(SPIRV_TOOLS_FOUND)
 endif()
 
 if(NOT SPIRV_TOOLS_SPIRV_AS)
-  message(WARNING "spirv-as not found! SPIR-V assembly tests will not be run.")
+  message(FATAL_ERROR "spirv-as not found!")
   set(SPIRV_TOOLS_SPIRV_AS_FOUND False)
 endif()
 
 if(NOT SPIRV_TOOLS_SPIRV_VAL)
-  message(WARNING "spirv-val not found! SPIR-V generated for test suite will not be validated.")
+  message(FATAL_ERROR "spirv-val not found!")
   set(SPIRV_TOOLS_SPIRV_VAL_FOUND False)
```

Comment:
Ditto for `SPIRV_TOOLS_SPIRV_VAL_FOUND` and `config.spirv_tools_have_spirv_val`.